dist/snap/snapcraft.yaml

```diff
@@ -15,7 +15,7 @@ apps:
       # Fallback to XWayland if running in a Wayland session.
       DISABLE_WAYLAND: 1
     desktop: usr/share/applications/input-leap.desktop
-    common-id: com.github.input-leap.input-leap
+    common-id: io.github.input_leap.InputLeap
```

Comment:
Why the previous value isn't good? We do not own input_leap.github.io or input_leap.github.com.

Reply:
because the previous value is not what appstream has agreed upon

Reply:
>"Each AppStream component has a unique identifier (the value of the <id/> tag), which is unique across all distributions" 
 our name for flathub has to be `io.github.input_leap.InputLeap`  so for snap it must also